src/wp-includes/rest-api/endpoints/class-wp-rest-templates-controller.php

```diff
@@ -525,6 +575,17 @@ protected function prepare_item_for_database( $request ) {
 			$changes->post_excerpt = $template->description;
 		}
 
+		if ( 'wp_template' === $this->post_type ) {
+			if ( isset( $request['is_wp_suggestion'] ) ) {
```

Comment:
These 2 conditionals can be combined as there's no other code within the first if block.